webpack.config.js

```diff
@@ -69,7 +69,7 @@ module.exports = {
     plugins: [
         new webpack.DefinePlugin({
             DIAGNOSTICS: true,
-            FAKE_SERVER: true
+            FAKE_SERVER: false
```

Comment:
This should still be `true`. The default for the client is to use the fake server. That way any dev being done here can occur in relative isolation from the server.